lang/src/OTS/IMP/org/partiql/ots/legacy/operators/StandardBinaryConcatOp.kt

```diff
@@ -17,7 +17,12 @@ object StandardBinaryConcatOp : BinaryConcatOp() {
     override val defaultReturnTypes: List<CompileTimeType> =
```

Comment:
This is getting into matters of taste, but I wouldn't bother neither about "Standard" nor "Legacy" in the names of the operator implementations (`StandardBinaryConcatOp` here) -- they are scoped inside the legacy plugin instance, which should be enough.